tests/Integration/Monitoring/Test_DataOperationSender.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+
+import time
+from DIRAC import gLogger
+from DIRAC.MonitoringSystem.Client.DataOperationSender import DataOperationSender
+
+gLogger.setLevel("INFO")
```

Comment:
```suggestion
gLogger.setLevel("DEBUG")
```
to see everything in the test.